config/bootstrap.php

```diff
@@ -4,14 +4,15 @@
 use Cake\Routing\DispatcherFactory;
 use CookieWarning\Routing\Filter\CookieWarningFilter;
 use CookieWarning\Validation\ConfigValidator;
-use Cake\Core\Configure\Engine\PhpConfig;
 
-Configure::config('default', new PhpConfig(CONFIG));
-Configure::load('cookie_warning', 'default', false);
+$config = Configure::read('cookie_warning');
```

Comment:
Same comment as above : the key should be CamelCased as other config keys in the base app skeleton are.